### PR TITLE
Create lang_fa.strings

### PR DESCRIPTION
**Pull request by Jav4dR**
Persian Language

### CODE REVIEW

File: Telegram/SourceFiles/langs/lang_fa.strings [hunk 1]
```diff
@@ -0,0 +1,482 @@
...
+"lng_weekday1" = "دوشنبه";
+"lng_weekday2" = "سه شنبه";
+"lng_weekday3" = "چهارشنبه";
+"lng_weekday4" = "پنج شنبه";
+"lng_weekday5" = "جمعه";
+"lng_weekday6" = "شنبه";
+"lng_weekday7" = "يکشنبه";
+
+"lng_weekday1_full" = "دوشنبه";
+"lng_weekday2_full" = "سه شنبه";
```
Comment (ahmadalli) [inline Telegram/SourceFiles/langs/lang_fa.strings]: سه‌شنبه

File: Telegram/SourceFiles/langs/lang_fa.strings [hunk 2]
```diff
@@ -0,0 +1,482 @@
...
+"lng_weekday3_full" = "چهارشنبه";
+"lng_weekday4_full" = "پنج شنبه";
```
Comment (ahmadalli) [inline Telegram/SourceFiles/langs/lang_fa.strings]: پنج‌شنبه

File: Telegram/SourceFiles/langs/lang_fa.strings [hunk 3]
```diff
@@ -0,0 +1,482 @@
...
+"lng_weekday5_full" = "جمعه";
+"lng_weekday6_full" = "شنبه";
+"lng_weekday7_full" = "يکشنبه";
```
Comment (ahmadalli) [inline Telegram/SourceFiles/langs/lang_fa.strings]: یک‌شنبه

**Comment by telegramdesktop** [comment]
Languages are not managed from here, sorry.

**Pull request closed without merging**